clevertap-core/src/main/java/com/clevertap/android/sdk/inapp/CTInAppNotificationMedia.java

```diff
@@ -126,4 +126,8 @@ public boolean isVideo() {
         String contentType = this.getContentType();
         return contentType != null && this.mediaUrl != null && contentType.startsWith("video");
     }
+
+    public boolean isStreamMedia() {
```

Comment:
Function names can be made same here and for `mediaIsStreamable()`

Reply:
done